src/transport_performance/gtfs/validation.py

```diff
@@ -273,8 +277,11 @@ def viz_stops(
 
         pre, ext = os.path.splitext(out_pth)
         if ext != ".html":
-            print(f"{ext} format not implemented. Writing to .html")
-            out_pth = os.path.normpath(pre + ".html")
+            raise ValueError(
+                f"{ext} format not implemented. Accepted "
+                "formats include ['.html'] . Try out_pth="
+                f"'{pre+'.html'}"
+            )
```

Comment:
What are your thoughts on making this a warning instead of an error? The previous implementation swapped the file extension to html, which was quite handy. Maybe we could just warn the user that this is happening instead of raising an error at this point?

Reply:
I'd be happy with this. Happy to apply these changes if you'd like?

Reply:
Resolving the conversation since this is resolve via commits [607158b](https://github.com/datasciencecampus/transport-network-performance/pull/137/commits/607158b3764a1736ad5cb5db262ef83bb7cdd0ca) and [f30c6c0](https://github.com/datasciencecampus/transport-network-performance/pull/137/commits/f30c6c0826093a4d3814bc5107e5f0c41202ab82).